app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
+end
```

Comment:
if you don't need a mailer, why maintain it?
Make sure to use the rails generators to help you although you should be able to evaluate what to keep and what you may remove.

Reply:
Sure, I used the generator but I forget to delete this code.